### DISCUSSION TITLE
Force i3's window decorations for every window.

### DISCUSSION THREAD

**Discussion by ra-om** [i3/i3#4999] [Q&A] [answered]
Hello,

Some programs (for example Steam and Nicotine) have their own forced decorations with unresponsive buttons, and i3's own don't appear unless I tab or stack them. Is there a way to force i3 to always show it's own window frames?

**Comment by Airblader** [comment, marked as answer]
```
for_window [all] border normal
```

This should do the trick as it is executed after mapping the window is negotiated.

**Reply by ra-om** [reply]
Yes it does, thank you!
